app/src/main/kotlin/com/simplemobiletools/calculator/activities/MoneyActivity.kt

```diff
@@ -32,6 +35,8 @@ class MoneyActivity : SimpleActivity(), Calculator , TaxCalculator {
     private var taxDialog:AlertDialog.Builder? = null
 
     lateinit var calc: MoneyCalculatorImpl
+    private var tipValue = 15
```

Comment:
What does this variable and why is it hard coded in the controller ?

Reply:
Good catch, that was from an earlier implementation that has since changed.

Reply:
Alright sounds good